helm/dagster/charts/dagster-user-deployments/values.yaml

```diff
@@ -155,6 +155,15 @@ serviceAccount:
 # Whether to bind a role to the service account that allows it to create Kubernetes jobs
 rbacEnabled: true
 
+extraInitContainers: []
```

Comment:
can we also add this to the dagster-user-deployments sections of the values.yaml in the main helm chart, for people who are not deploying the subchart separately? https://github.com/dagster-io/dagster/blob/master/helm/dagster/values.yaml#L276